### ISSUE TITLE
BC-Container Could not download reference symbols

### ISSUE THREAD

**Issue by CarlosAndreuLlaneras** [microsoft/navcontainerhelper#1779]
Hi,
I am having some problems downloading symbols from the dockers that I am creating during the last week.
From previous dockers I can download correctly but in the new ones, it shows folowing message: 
```
_The request for path /navserverdev/dev/packages?publisher=Microsoft&appName=System&versionText=1.0.0.0&tenant=default failed with code NotFound. Reason: Not Found
The request for path /navserverdev/dev/packages?publisher=Microsoft&appName=Application&versionText=18.0.0.0&tenant=default failed with code NotFound. Reason: Not Found_

``` 

This problem is independent on the version of Business Central that I am creating. The same thing happens to me with a BC18, than with a BC16.4

To reproduce it:
- I have created a new virtual machine with "trafik" enabled with the latest version 18 (18.0.23023). (the virtual machine has been created using the azure template (http://aka.ms/getbcext))
- I have created a new App from VSCode using "AL: Go!"
- I have configured launch.json against the new docker and when I try to download symbols the error message is the following:

![image](https://user-images.githubusercontent.com/53298196/110515629-e06ea080-8108-11eb-9dc8-069c71f886e2.png)

![image](https://user-images.githubusercontent.com/53298196/110515712-f67c6100-8108-11eb-8b26-4e68b2b73de1.png)

I am missing something?
Any help will be welcome.
Thanks

```
BCContainerHelper is version 2.0.6
```

**Comment by freddydk** [comment]
What version of the vsix are you running?
Could you try downloading the one from the container and installing that?

**Comment by CarlosAndreuLlaneras** [comment]
Hi,
I've updated vsix from the container. Now the version is 7.0.427793 and the error message is the same.
![image](https://user-images.githubusercontent.com/53298196/110542615-8f22d900-8129-11eb-9979-6c6336aa1647.png)
thanks

**Comment by freddydk** [comment]
2.0.6 does have a bug when creating a container using traefik.
The fix was added to 2.0.7 preview on March 2nd here: https://github.com/microsoft/navcontainerhelper/commit/008718184b09e3e2742734fe891af7af3e41661b#diff-12bc248f44906c327ddb7b8364902dd4d252c4fb09ee350a53710d1d668ee25a

You can either apply the same changes to your new-bccontainer in c;\program files\windowspowershell\modules\bccontainerhelper\2.0.6\containerhandling

or you can install the bccontainerhelper preview

and then recreate the container.
I expect 2.0.7 to be released non-preview later this week.

**Comment by CarlosAndreuLlaneras** [comment]
Thanks freddy,
I've update to BCContainerHelper 2.0.7 preview and it has fixed the problem!!!!

Thanks a lot.

P.S.: If someone else needs to install the preview version:
`Install-Module BcContainerHelper -force -allowPrerelease`

Regards
Carlos
